### ISSUE TITLE
grpo微调Qwen3-VL-8B-Instruct，跑的示例代码rewards/MathAccuracy/mean一直为0，rewards/Format/mean不为0

### ISSUE THREAD

**Issue by l878619717** [modelscope/ms-swift#6660]
**Describe the bug**
What the bug is, and how to reproduce, better with screenshots(描述bug以及复现过程，最好有截图)
用的L20 2个卡grpo微调Qwen3-VL-8B-Instruct，跑的示例代码ms-swift-main\examples\train\grpo\internal\vllm_vl7b.sh。rewards/MathAccuracy/mean一直为0，rewards/Format/mean不为0。
MAX_PIXELS=1003520 \
CUDA_VISIBLE_DEVICES=0,1 \
NPROC_PER_NODE=2 \
swift rlhf \
    --rlhf_type grpo \
   --model /mnt/models/Qwen3-VL-4B-Instruct \
   --train_type lora \
   --dataset /mnt/data/chartqa_digit_r1v_format \
   --load_from_cache_file true \
   --use_vllm true \
   --vllm_mode colocate \
   --vllm_gpu_memory_utilization 0.5 \
   --vllm_max_model_len 16384 \
  --vllm_tensor_parallel_size 2 \
  --torch_dtype bfloat16 \
  --system examples/train/grpo/prompt.txt \
  --num_train_epochs 1 \
  --per_device_train_batch_size 1 \
  --per_device_eval_batch_size 1 \
  --learning_rate 1e-4 \
  --save_total_limit 2 \
  --logging_steps 5 \
  --output_dir output \
  --gradient_accumulation_steps 2 \
  --warmup_ratio 0.05 \
  --dataloader_num_workers 1 \
  --max_completion_length 512 \
  --reward_funcs accuracy format \
  --num_generations 4 \
  --sleep_level 1 \
  --temperature 1.0 \
  --offload_optimizer true \
  --offload_model true \
  --top_p 0.85

**Comment by hjh0119** [comment]
```diff
- MAX_PIXELS=1003520 \
+ IMAGE_MAX_TOKEN_NUM=1536 \
```

Set `log_completions true` to view detailed rollout completions.

**Comment by l878619717** [comment]
Set `log_completions true` 后有详细completion，测试了奖励函数对某一个completion有奖励输出，但是"MathAccuracy": [0.0, 0.0, 0.0, 0.0] 数组却是0，好奇怪



> - MAX_PIXELS=1003520 \
> + IMAGE_MAX_TOKEN_NUM=1536 \
> Set `log_completions true` to view detailed rollout completions.

**Comment by l878619717** [comment]
> Set `log_completions true` 后有详细completion，测试了奖励函数对某一个completion有奖励输出，但是"MathAccuracy": [0.0, 0.0, 0.0, 0.0] 数组却是0，好奇怪
> 
> > * MAX_PIXELS=1003520 \
> > 
> > 
> > * IMAGE_MAX_TOKEN_NUM=1536 
> >   Set `log_completions true` to view detailed rollout completions.

用的库里面的'accuracy': MathAccuracy奖励

**Comment by l878619717** [comment]
使用--external_plugins参数重新指定文件路径就可以了，函数改个名就可以了
